From 725cb07d90c7949a971378635e7755ff9a54d25d Mon Sep 17 00:00:00 2001
From: Andy Yan <andy.yan@rock-chips.com>
Date: Wed, 20 Dec 2023 15:59:23 +0800
Subject: [PATCH] clk: rockchip: rk3036: Add 1000 MHZ cpu clk rate

Set armclk to 1000 MHZ, so mac_clk_ref can get 50 MHZ
from it to work well.

Signed-off-by: Andy Yan <andy.yan@rock-chips.com>
---
 drivers/clk/rockchip/clk-rk3036.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/rockchip/clk-rk3036.c b/drivers/clk/rockchip/clk-rk3036.c
index d341ce0708aac3..d789a09b35d080 100644
--- a/drivers/clk/rockchip/clk-rk3036.c
+++ b/drivers/clk/rockchip/clk-rk3036.c
@@ -96,6 +96,7 @@ static struct rockchip_pll_rate_table rk3036_pll_rates[] = {
 	}
 
 static struct rockchip_cpuclk_rate_table rk3036_cpuclk_rates[] __initdata = {
+	RK3036_CPUCLK_RATE(1000000000, 4),
 	RK3036_CPUCLK_RATE(816000000, 4),
 	RK3036_CPUCLK_RATE(600000000, 4),
 	RK3036_CPUCLK_RATE(312000000, 4),
